### ISSUE TITLE
Subtitles

### ISSUE THREAD

**Issue by tomhvrdy** [braz96/skin.mimic#57]
I haven't been using Kodi for long, and I just recently stumbled onto this skin which is totally gorgeous! I'm having quite the problem though - I use subtitles quite a lot, and in the default skin I'm able to disable them when they're incorrect or offset the timing so that it matches the audio. However, on Mimic when I try to go to the subtitles submenu when a show is playing, it only brings up a dialogue to change the SRT file I'm using, and even then I don't seem to be able to select anything in the dialogue - I can only back out of it. Am I totally missing a setting or add-on somewhere, or is this something that needs to be fixed on your end?

**Comment by braz96** [comment]
What version of Mimic are you using? Those options are in the video
settings dialog in older versions and were also added to player controls in
newer versions.

Every skin has an video osd dialog, that's the official home for subtitle
settings.

**Comment by tomhvrdy** [comment]
I have no idea what version I'm using, I can't find where it says anywhere? I've looked in the video settings dialog and it doesn't have anything at all about subtitles - and like I said, when I click on the subtitle option in player controls it only pulls up a dialog to change which SRT file I'm using.

It might be something weird with my Kodi, it's been acting up and when I changed from Mimic to other skins I still couldn't find those options anymore. I'll just have to mess around with it I guess, but thanks!
